CadRevealComposer/Utils/Simplify.cs

```diff
@@ -129,4 +130,27 @@ public static List<APrimitive> OptimizeVertexCountInMeshes(IEnumerable<APrimitiv
 
         return processedGeometries;
     }
+
+    public static Mesh ConvertToConvexHull(Mesh inputMesh, float tolerance = 1.0E-1f)
+    {
+        // Build vertex list to hand to the convex hull algorithm
+        var meshVertices = inputMesh.Vertices.Select(v => new double[] { v.X, v.Y, v.Z });
+
+        // Create the convex hull
+        var convexHullOfMesh = ConvexHull.Create(meshVertices.ToArray(), tolerance);
+        if (convexHullOfMesh.ErrorMessage.Length != 0)
```

Comment:
I guess there is a better way to cehck if we have an error. Kinda like convexHullOfMesh.status or similar?